### ISSUE TITLE
Unable to change room

### ISSUE THREAD

**Issue by sledmonkey** [NorthernMan54/homebridge-cmd-television#18]
Aside from the inability to change the icon of the apple tv, i'm also unable to change rooms via the Home app in iOS (changing via Home in macOS works).  It keeps reverting back to "Equipment Room" which is where all my new devices start.

**Comment by NorthernMan54** [comment]
Yeah I keep getting hit with that as well.  To resolve would need a massive rework of the plugin which may not be worth the effort.

But to reduce the occurrence of the issue, try to avoid restarting homebridge.
